packages/coq/coq.8.16.1/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+maintainer: "coqdev@inria.fr"
+authors: "The Coq development team, INRIA, CNRS, and contributors."
+homepage: "https://coq.inria.fr/"
+bug-reports: "https://github.com/coq/coq/issues"
+dev-repo: "git+https://github.com/coq/coq.git"
+license: "LGPL-2.1-only"
+synopsis: "Formal proof management system"
+description: """
+The Coq proof assistant provides a formal language to write
+mathematical definitions, executable algorithms, and theorems, together
+with an environment for semi-interactive development of machine-checked
+proofs. Typical applications include the certification of properties of programming
+languages (e.g., the CompCert compiler certification project and the
+Bedrock verified low-level programming library), the formalization of
+mathematics (e.g., the full formalization of the Feit-Thompson theorem
+and homotopy type theory) and teaching.
+"""
+
+depopts: [
+  "coq-native"
+]
+depends: [
+  "ocaml" {>= "4.09.0"}
+  "ocamlfind" {>= "1.8.1"}
+  "dune" {>= "2.5.1"}
+  "conf-findutils" {build}
+  "zarith" {>= "1.11"}
+]
+conflicts: [
+  "base-nnp"
```

Comment:
Are there any news on the OCaml 5.0 support by any chance?

Reply:
@kit-ty-kate no updates for a while to my knowledge, we still have two critical TODOs here: https://github.com/coq/coq/projects/55

Reply:
Is there any way I can help?

Reply:
I will pass on the offer of help to the Coq Zulip chat (there is an OCaml 5.0 topic there), and I guess we'll see what happens.